wwwroot/src/pages/Landing.js

```diff
@@ -0,0 +1,260 @@
+    },
+    '&:last-child > div': {
+      width: theme.spacing(16),
+      position: 'relative',
+      top: -theme.spacing(1)
+    }
+  },
+  landing2ImgContainer: {
+    width: theme.spacing(15),
+    height: theme.spacing(15),
+    margin: `0 auto ${theme.spacing(2.5)}px`
+  },
+  test: {backgroundColor: 'red'}
+}))
+
+
+function Landing1() {
+  const classes = useStyles();
+
+  return (
+    <Container
+      className={classes.landing1root}
+      maxWidth='md'
+    >
+      <Container className={classes.landing1CopyContainer}>
+        <Typography className={classes.landing1CopyTitle} component='h1'>
+          {landing1Data.title}
+        </Typography>
+
+        <Typography className={classes.landing1CopyBody}>
+          {landing1Data.copy}
+        </Typography>
+
+        <Box 
+          className={classes.landing1ButtonCollection} 
+          display='flex'
+          justifyContent="flex-end"
+        >
+          <Button
+            variant='outlined'
+            color='primary'
+            component={RouterLink}
+            to='/why-us'
+          >
+            <Typography>
+              Learn more
+            </Typography>
+          </Button>
+
+          <Button
+            variant='contained'
+            color='primary'
+            component={RouterLink}
+            to='/signin'
+          >
+            <Typography>
+              Get Started
+            </Typography>
+          </Button>
+        </Box>
+      </Container>
+
+      <img src={triangleChild} alt='child reading' className={classes.landing1TriangleChild}/>
+      <img src={circleChild} alt='child at computer' className={classes.landing1CircleChild}/>
+      <img src={squareChild} alt='child reading tablet' className={classes.landing1SquareChild}/>
+
+    </Container>
+  )
+}
+
+
+
+export function Landing2() {
+  const classes = useStyles();
+
+  return (
+    <Container
+      className={classes.landing2root}
+      maxWidth='lg'
+    >
+      <Container className={classes.landing2Container}>
+        <Typography className={classes.landing2Title} component='h2'>
+          {landing2Data.title}
+        </Typography>
+
+        <Grid
+          container
+          justify='space-between'
+          spacing={3}
+        >
+          {landing2Data.icons.map((i, index) => (
+            <Grid 
+              item
+              key={index} 
+              className={classes.landing2ItemBox}
+              xs={12}
+              sm={4}
+            >
+              <Box className={classes.landing2ImgContainer}>
+                <img src={i.img}/>
+              </Box>
+
+              <Typography component='h3'>
+                {i.title}
+              </Typography>
+
+              <Typography>
+                {i.copy}
+              </Typography>
+            </Grid>
+          ))}
+        </Grid>
+      </Container>
+    </Container>
+  )
+}
+
+export default function Landing() {
+  return (
+    <Layout>
+      <Landing1/>
```

Comment:
Can you please rename the elements from landing1 and landing2 to something more meaningful?

Reply:
changed landing1 to landingHome (I don't know what else to call this sorry)
changed landing2 to featureCollection